src/app/Instance/InstancesListPage/InstancesListPage.tsx

```diff
@@ -63,53 +63,56 @@ const InstancesListPage = (): JSX.Element => {
     [t]
   );
 
-  const columnNames = [
-    {
-      accessor: "name",
-      label: t("common.name"),
-      formatter: (value: IRowData, row?: IRow): JSX.Element => {
-        const bridgeId = (row as BridgeResponse)?.id ?? "";
-        const status = (row as BridgeResponse)?.status;
+  const columnNames: TableColumn[] = useMemo(
+    () => [
+      {
+        accessor: "name",
+        label: t("common.name"),
+        formatter: (value: unknown, row?: IRowData): JSX.Element => {
+          const bridgeId = (row as BridgeResponse)?.id ?? "";
+          const status = (row as BridgeResponse)?.status;
 
-        return canEditResource(status) ? (
-          <Link
-            data-testid="tableInstances-linkInstance"
-            to={`/instance/${bridgeId}`}
-          >
-            {value}
-          </Link>
-        ) : (
-          <>{value}</>
-        );
+          return canEditResource(status) ? (
+            <Link
+              data-testid="tableInstances-linkInstance"
+              to={`/instance/${bridgeId}`}
+            >
+              {value}
+            </Link>
+          ) : (
+            <>{value}</>
+          );
+        },
       },
-    },
-    {
-      accessor: "submitted_at",
-      label: t("common.submittedAt"),
-      formatter: (value: IRowData): string => {
-        const date = new Date(value as unknown as string);
-        return formatDistance(date, new Date()) + " " + t("common.ago");
+      {
+        accessor: "submitted_at",
+        label: t("common.submittedAt"),
+        formatter: (value: unknown): string => {
+          const date = new Date(value as string);
+          return formatDistance(date, new Date()) + " " + t("common.ago");
+        },
       },
-    },
-    {
-      accessor: "status",
-      label: t("common.status"),
-      formatter: (value: IRowData, row?: IRow): JSX.Element => {
-        const statusString = value as unknown as ManagedResourceStatus;
-        const requestedAt = new Date(
-          (row as BridgeResponse)?.modified_at ??
-            (row as BridgeResponse)?.submitted_at
-        );
-        return (
-          <SEStatusLabel
-            status={statusString}
-            resourceType={"bridge"}
-            requestedAt={requestedAt}
-          />
-        );
+      {
+        accessor: "status",
+        label: t("common.status"),
+        formatter: (value: unknown, row?: IRowData): JSX.Element => {
+          const statusString = value;
+          const requestedAt = new Date(
+            (row as BridgeResponse)?.modified_at ??
+              (row as BridgeResponse)?.submitted_at
+          );
+          return (
+            <SEStatusLabel
```

Comment:
@vpellegrino how often is updated SEStatusLabel? It seems that when the status is changed than all cells in the row are generated again. So cypress lost reference for the visible element.

I am not sure if it could be caused any callBack. Previously only the value was changed not whole element.

Reply:
Columns definitions (so formatters) are already memoized (useMemo above), so I think there is nothing we can do from that point of view.
Is it possible to write tests differently?

Reply:
I have checked that:
A. this branch has similar Cypress version as main branch => probably no bug in cypress.
B. PF Table changes similar elements and their attributes as ResponsiveTable => See:
![image](https://user-images.githubusercontent.com/1536609/201715505-ac905dc8-ba9e-4341-aba7-1873fc970044.png)

I am not sure what could cause so many detached DOM element errors. I need to find a way how to rewrite selectors for affected tests. I have tried same logic as @jomarko and it is failing.

```
    cy.ouiaId("Instances list table", "PF4/Table")
      .ouiaId(newInstanceName, "PF4/TableRow")
      .ouiaId("creating", "PF4/Button")
      .should("have.text", "Creating");
    cy.ouiaId("Instances list table", "PF4/Table")
      .ouiaId(newInstanceName, "PF4/TableRow")
      .ouiaId("creating", "PF4/Button")
      .click();
```

Reply:
@vpellegrino I agree with you. I have checked it as a user and ResponsiveTable is similar to PF4 from a user's point of view. Is possible to use the OUIA standard in selector.

The big disadvantage of this approach is that we lost the reliability of our tests. If I cannot click on the element  by
```
    cy.ouiaId("Instances list table", "PF4/Table")
      .ouiaId(newInstanceName, "PF4/TableRow")
      .ouiaId("creating", "PF4/Button")
      .click();
```
so we need to change the structure of our tests.

Reply:
@sjamboro, can you please help me by providing a PR fixing those tests? Thank you very much.

Reply:
@vpellegrino yes of course. I tried ito fix it yesterday. Unfortumately without any result.